index.js

```diff
@@ -17,18 +19,17 @@ function Redis (host, port, pass, db = 0) {
   this.pass = pass;
   this.db = db;
   this.ready = false;
-  this.connect();
```

Comment:
curious: why not connect here?

Reply:
the constructor cannot be async

Reply:
Aha, OK.. I'll need to take note of this when upgrading the repos .. do you think you can update READMEs for these packages with their use cases?